daft/expressions/expressions.py

```diff
@@ -1461,7 +1913,23 @@ def left(self, nchars: int | Expression) -> Expression:
         """Gets the n (from nchars) left-most characters of each string
 
         Example:
-            >>> col("x").str.left(3)
+            >>> import daft
+            >>> df = daft.from_pydict({"x": ["foo", "bar", "baz"]})
```

Comment:
Lets have strings that are more than `n` chars long, exactly `n` chars, and less than `n` chars.